docs/references.txt

```diff
@@ -62,6 +62,7 @@
 .. _CTAO: https://www.ctao.org/
 .. _H.E.S.S.: https://www.mpi-hd.mpg.de/hfm/HESS/
 .. _HAWC: https://www.hawc-observatory.org/
+.. _SWGO: https://www.swgo.org/
```

Comment:
At some point, we should order this list in some logical fashion (eg: Alphabetical/ Detector type/etc)